kong/clustering/compat/checkers.lua

```diff
@@ -54,6 +54,57 @@ local compatible_checkers = {
         end
       end
 
+      -- Support legacy queueing parameters for plugins that used queues prior to 3.3.  `retry_count` has been
+      -- completely removed, so we always supply the default of 10 as that provides the same behavior as with a
+      -- pre 3.3 CP.  The other queueing related legacy parameters can be determined from the new queue
+      -- configuration table.
+      for _, plugin in ipairs(config_table.plugins or {}) do
+        local config = plugin.config
+        if plugin.name == 'statsd' or plugin.name == 'datadog' then
```

Comment:
@hbagdi Do we want to support potential custom plugins?

Reply:
We do not support custom plugins here. We will get there some day with dynamic schema diff but that is not on the near term horizon yet.